### DISCUSSION TITLE
[Feature] only backs up new photos

### DISCUSSION THREAD

**Discussion by GodD6366** [immich-app/immich#12436] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I plan to migrate from Synology Photos to immich. First, I will use immich-cli to migrate the already backed-up images, and then start backing up new photos from the mobile app. However, it seems that immich does not support this configuration. It is recommended to refer to Synology Photos and add this feature, so that it can skip the process of downloading original images from iCloud and checking for duplicates with immich by comparing hashes.

![截屏 2024-09-07 21 02 28](https://github.com/user-attachments/assets/a498bf01-4d0e-4027-9b54-3c7920748eb9)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4169
